CHANGES.md

```diff
@@ -1,6 +1,12 @@
 ringpop-go changes
 ==================
 
+v0.x.y (to be released)
```

Comment:
I also did this, see:
https://github.com/uber/ringpop-go/pull/118/files#diff-8b1c3fd0d4a6765c16dfd18509182f9dR4

I called it "0.3.1", as I intend it to be in the next _minor_ release.